nuxt.config.ts

```diff
@@ -12,15 +18,25 @@ export default defineNuxtConfig({
         path: 'pathe',
       },
     },
+    server: {
+      headers: crossOriginHeaders,
+    },
   },
   nitro: {
     routeRules: {
       '/**': {
-        headers: {
-          // cross origin isolation is required since oxc-parser uses shared array buffer
-          'Cross-Origin-Embedder-Policy': 'require-corp',
-          'Cross-Origin-Opener-Policy': 'same-origin',
-        },
+        headers: crossOriginHeaders,
+      },
+    },
+    vercel: {
+      config: {
+        routes: [
+          {
+            src: '^/_nuxt/(.*)$',
+            // @ts-ignore
+            headers: crossOriginHeaders,
+          },
+        ],
```

Comment:
I just tried something random and it seems to work :smile:
Do you know if this is correct? Should we check with nuxt/nitro folks about the proper solution?